src/main.rs

```diff
@@ -99,7 +99,7 @@ impl ExtrinsicOpts {
 }
 
 #[derive(Clone, Copy, Debug)]
-#[cfg_attr(test, derive(PartialEq))]
+#[cfg_attr(any(test, feature = "binaryen-as-dependency"), derive(PartialEq))]
```

Comment:
Why do we only derive `PartialEq` under these features?

Reply:
Only needed if either of these are enabled.

Reply:
Yeah sure but why be stingy and not give it to everyone?